ank/src/cli_commands/get_agents.rs

```diff
@@ -49,19 +50,28 @@ impl CliCommands {
 }
 
 fn transform_into_table_rows(
-    agents_map: impl Iterator<Item = String>,
+    agents_map: impl Iterator<Item = (String, FilteredAgentAttributes)>,
     workload_states_map: &WorkloadStatesMap,
 ) -> Vec<AgentTableRow> {
     let mut agent_table_rows: Vec<AgentTableRow> = agents_map
-        .map(|agent_name| {
+        .map(|(agent_name, agent_attributes)| {
             let workload_states_count = workload_states_map
                 .get_workload_state_for_agent(&agent_name)
                 .len() as u32;
-
-            AgentTableRow {
+            let mut agent_table_row = AgentTableRow {
                 agent_name,
                 workloads: workload_states_count,
+                cpu_load: "".to_string(),
+                free_memory: "".to_string(),
```

Comment:
Why defaults (without default) and then overwriting? We can assign directly here